families/families.js

```diff
@@ -9,36 +10,17 @@ logoutButton.addEventListener('click', () => {
     logout();
 });
 
-function displayFamilies() {
+export async function displayFamilies() {
     // fetch families from supabase
-
+    const families = await getFamilies();
     // clear out the familiesEl
-
+    familiesEl.textContent = '';
     for (let family of families) {
-        // create three elements for each family, one for the whole family, one to hold the name, and one to hold the bunnies
-        // your HTML Element should look like this:
-        // <div class="family">
-        //    <h3>the Garcia family</h3>
-        //    <div class="bunnies">
-        //        <div class="bunny">Fluffy</div>
-        //        <div class="bunny">Bob</div>
-        //    </div>
-        // </div>
-        // add the bunnies css class to the bunnies el, and family css class to the family el
-        // put the family name in the name element
-        // for each of this family's bunnies
-        //    make an element with the css class 'bunny', and put the bunny's name in the text content
-        //    add an event listener to the bunny el. On click, delete the bunny, then refetch and redisplay all families.
-        // append this bunnyEl to the bunniesEl
+        familiesEl.append(await renderFamily(family));
     }
-
-    // append the bunniesEl and nameEl to the familyEl
-
-    // append the familyEl to the familiesEl
 }
 
 window.addEventListener('load', async () => {
     const families = await getFamilies();
-
     displayFamilies(families);
```

Comment:
Just a quick note here for something to pay attention to in the future. On this line you're calling displayFamilies with the argument of families. On line 12 where you write the function displayFamilies does not take any argument. I think this one ends up not giving any errors because one of the first things you do in that function is get the families and so it pulls any of that updated data that the user submits and writes to the database. Like I said, just something to pay attention to when reviewing, little things like this may or may not cause errors down the line.